caikit/runtime/model_management/model_loader.py

```diff
@@ -94,14 +87,9 @@ def load_model(
         # Set up the async loading
         args = (local_model_path, model_id, model_type)
         log.debug2("Loading model %s async", model_id)
-        if aborter is not None:
-            log.debug3("Using abortable action to load %s", model_id)
-            action = AbortableAction(aborter, self._load_module, *args)
-            future_factory = partial(self._load_thread_pool.submit, action.do)
-        else:
-            future_factory = partial(
-                self._load_thread_pool.submit, self._load_module, *args
-            )
+        future_factory = partial(
```

Comment:
Ok, I think I understand what's going on here: This class will always be used _inside_ an `AbortableContext` (that may or may not actually be abortable, depending on whether the `aborter`/`interrupter` are real instances).

Reply:
Yeah, from some tracing I did it seemed like `aborter` was never even set here, and if it was then it would have been double-wrapping with destroyable threads